cogs/assistance.py

```diff
@@ -721,7 +721,7 @@ async def emureco(self, ctx, console=None):
         if self.check_console(console, ctx.message.channel.name, '3ds'):
             embed = discord.Embed(title="EmuNAND for 3DS", color=0xe60012)
             embed.description = cleandoc("""
-            With the recent advances in hacking methods and safety, it is no longer recommended to use an emuNAND on a 3DS/2DS system.
+            Since 2016 it has no longer been recommended to use an emuNAND on a 3DS/2DS system.
```

Comment:
"As of 2016, it is no longer recommended to use an emuNAND on a 3DS/2DS system."